pkg/server/server_test.go

```diff
@@ -343,3 +334,7 @@ func testWebhookRouter(t *testing.T, bolt storage.ServiceStorage) *router.Webhoo
 
 	return webhookRouter
 }
+
+func is2xxResponse(statusCode int) bool {
+	return statusCode/100 == 2
+}
```

Comment:
We have this defined in multiple places, perhaps it's time to drop this into util?

Reply:
done